docs/reference/release-notes/v2.28/index.md

```diff
@@ -0,0 +1,98 @@
+---
+date: "2020-12-01"
+version: "2.28.0"
+---
+
+# [v2.28](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) (December 2020 #1)
+
+---
+
+Welcome to `gatsby@2.28.0` release (December 2020 #1).
+
+Key highlights of this release:
+
+- [New `gatsby new`](#new-gatsby-new) - new, interactive way to create a Gatsby site
+- [Improved `react-refresh` integration](#improved-react-refresh-integration)
+- [Feature flags in gatsby-config](#feature-flags-in-gatsby-config) - set your feature toggles without environment variables
+- [Experimental: Lazy images in develop](#experimental-lazy-images-in-develop) - run image transformations only when they are needed by browser
+- [Removed experimental lazy page bundling](#removed-experimental-lazy-page-bundling)
+- [Notable bugfixes](#notable-bugfixes)
+
+Sneak peek to next releases:
+
+- [Less aggressive cache invalidation](#less-aggressive-cache-invalidation)
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](../v2.27/index.md)<br>
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0)
+
+## new `gatsby new`
+
+In previous release we added [interactive way of scaffolding new gatsby project](../v2.27/index.md#create-gatsby) (with `npm init gatsby`). Now we adjusted `gatsby new` command when no arguments are passed to also use the same flow.
+
+## Improved `react-refresh` integration
+
+Gatsby had some support for `react-refresh` (or Fast Refresh as it can be referred to) already. This release improves it and adds better error overlays to be able to fix runtime and compilation error faster!
```

Comment:
```suggestion
Gatsby had some support for `react-refresh` (or Fast Refresh as it can be referred to) already. This release improves it and adds better error overlays to be able to fix runtime and compilation errors faster! Time = money, and we're giving you time back by helping you solve your errors more quickly!
```